mdocfile/section_data.py

```diff
@@ -104,5 +104,9 @@ def to_string(self):
                 v = ' '.join(str(el) for el in v)
             if v == 'nan':
                 v = 'NaN'
+            if isinstance(v, datetime):
+                v = v.strftime('%y-%b-%d  %H:%M:%S')
+            if isinstance(v, Path):
+                v = fr'X:\spoof\frames\{v.name}'
```

Comment:
should probably check for specific keys `DateTime` and `SubFramePath` here rather than just types, what do you think?

Reply:
oh yeah, that seems more reasonable :P

Reply:
Though on the other hand, I suppose if one breaks the others would as well...

Reply:
nah, I wouldn't expect keys to change, we might end up with another entry containing something of the same type though :)

Reply:
Sorry, I didn't explain myself: If one key with `datime` type breaks, I assume they will all break because of the same reason. I expect future `datetime` keys to use the same format as this one. The path also broke because of slashes instead of backslashes... Though I guess different kinds of paths should be handled differently, so maybe that should be key-based.

So I'm ok either way, but I think the `datetime` should stay type-based!

Reply:
Feels like a strong assumption for IMOD though, which has two conventions for defining simple 2D image rotation angles 😆 I'm fine either way too and don't expect it to break

Agree that the path should not be modified here